Cargo.toml

```diff
@@ -20,9 +21,13 @@ multihash-codetable = "0.1"
 quick-protobuf = "0.8"
 smallvec = "1"
 thiserror = "1"
+tracing = "0.1"
 unsigned-varint = "0.8"
 void = "1"
 
+# Explicitly specify the `curve25519-dalek` version which fixes SIMD build issues
+curve25519-dalek = "4.1.2"
+
```

Comment:
wouldn't just `cargo update` do the job?

Reply:
You mean with the same way we did with the `log`?

Reply:
I mean just running cargo update to update the lockfile

Reply:
https://github.com/eigerco/lumina/pull/216 It seems to work on lumina

Reply:
done